### ISSUE TITLE
LaTeX Error: Too deeply nested. => \tightlist 

### ISSUE THREAD

**Issue by idelsink** [idelsink/mexdown#15]
LaTeX Error: Too deeply nested. 

The list that it applies to:
```tex
\tightlist 
```

This can be solved by increasing the max nesting value in the `lists.sty` file in the `tex` directory.

**Comment by craigbass76** [comment]
I can't find such a file. Is it somewhere else now? I've been grepping any sty files for nesting, but nothing's popping up that looks likely yet.

**Comment by idelsink** [comment]
Yeah it's in the following location: https://github.com/idelsink/mexdown/blob/master/tex/latex/local/mexdown/packages/lists/lists.sty
